Python/bytecodes.c

```diff
@@ -2955,18 +2954,35 @@ dummy_func(
             PyCodeObject *code = (PyCodeObject *)func->func_code;
             DEOPT_IF(code->co_argcount != argcount, CALL);
             DEOPT_IF(!_PyThreadState_HasStackSpace(tstate, code->co_framesize), CALL);
+        }
+
+        op(_INIT_CALL_PY_EXACT_ARGS, (method, callable, args[oparg] -- new_frame: _PyInterpreterFrame*)) {
```

Comment:
It feels weird to push `new_frame`, but I do understand that it's sort of our only way to do this move.

If we didn't need to sneak a `SET_IP` in there during tracing, then `_PUSH_FRAME` could have just been bundled into this uop, right? Hm.

Reply:
Definitely weird (and I had to fix a few nits in the code generator to make it work) but I really want to have `_PUSH_FRAME` as a separate uop:

- There are a bunch of other instructions ending in `DISPATCH_INLINED()` and those will all be able to share `_PUSH_FRAME`.
- This uop needs to be special-cased by the (yet to be written) Tier 2 optimizer -- it's better to have fewer special cases.